tools/tune/drc/drc_gen_coefs.m

```diff
@@ -0,0 +1,117 @@
+function coefs = drc_gen_coefs(params, sample_rate);
+
+pkg load control;
```

Comment:
The Octave specifics those would result to error when running in Matlab are best to be placed inside 

```
if exist('OCTAVE_VERSION', 'builtin')
end
```